### ISSUE TITLE
Trailing comma for function calls

### ISSUE THREAD

**Issue by xorik** [FriendsOfPHP/PHP-CS-Fixer#5655]
## Rule request

Hi. I didn't find a rule that fixes trailing comma in function calls.

### Code snippet that shows how the rule would fix PHP code

  ```php
  <?php
  something(
      $foo,
      $bar
  );
  ```

Should become

  ```php
  <?php
  something(
      $foo,
      $bar,
  );
  ```

**Comment by keradus** [comment]
would you like to implement it, @xorik ?

**Comment by kubawerlos** [comment]
@keradus the topic is on hold here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4238 - since there is no conclusion if it should be one fixer for trailing comma in multiline or 3 separate fixers.

**Comment by keradus** [comment]
@kubawerlos , is this closed with merge of #4238 ?

**Comment by kubawerlos** [comment]
@keradus yes
